### ISSUE TITLE
How to calculate best mix of Eco and Pro plots

### ISSUE THREAD

**Issue by darkenbula** [drnick23/drplotter#29]
Hi DrNick

Is there a way to calculate this best mix of pro and eco plots?

For example i have approx 250TiB for an effective 401TiB with madmax @ C17

How would i calculate the best mix of these to effectively use either a 3090 or 4090 so i can decide if purchasing a GPU is worth it for me

**Comment by drnick23** [comment]
(Pro4x TiB)+(Eco3x Tib)=250 (your capacity)
(Pro4x Tib)/200 + (Eco3x Tib)/250 = 1 (1 4090 gpu)

Now you can solve like you would for:
x + y = 250
x/200 + y/250 = 1
where x is Pro4x and y is Eco3x

You'll get 160 Pro4x and 90 Eco3x.

160 Pro4x = 80% gpu utilization, with 657TiB effective
90 Eco3x = 20% gpu utilization, with 270TiB effective

So you would total 927TiB effective, which is 526 more eTib than C17 at the cost of a 4090, with approx 330W more power usage (although you need to subtract the gpu usage on C17 from this assuming it's not 0, let's say it's 30W).

In summary:
C17                 =  401 eTib
Eco3x + Pro3x = 927 eTib + cost 4090 + ~300W
